### PR TITLE
update models section to reflect renamed store methods

### PR DESCRIPTION
**Pull request by csantero**
Based on the changes in https://github.com/emberjs/data/pull/3235 and https://github.com/emberjs/data/pull/3241

### CODE REVIEW

**Comment by HeroicEric** [comment]
there's another reference at https://github.com/emberjs/guides/blob/master/source/models/handling-metadata.md

**Comment by csantero** [comment]
@HeroicEric Thanks, I've pushed a new diff with that change

File: source/models/finding-records.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 The Ember Data store provides a simple interface for finding records of a single
 type through the `store` object's `find` method. Internally, the `store`
-uses `find`, `findAll`, and `findQuery` based on the supplied arguments.
+uses either `findAll` or `findRecord` based on the supplied arguments.
 
 The first argument to `store.find()` is always the record type. The optional second
-argument determines if a request is made for all records, a single record, or a query.
+argument determines if a request is made for all records or just a single record.
```
Comment (tomdale) [inline source/models/finding-records.md]: Isn't this behavior deprecated and people should use `findAll`? @bmac

Comment (HeroicEric) [inline source/models/finding-records.md]: Those are being updated separately https://github.com/emberjs/guides/pull/346

**Comment by trek** [comment]
cc @igorT

**Comment by michaelrkn** [comment]
I think this should get merged in so that #346 can move forward with finishing the job (it already needs rebasing). @locks or @rwjblue can you do a final sanity check on this?

**Comment by locks** [comment]
:+1:

**Pull request merged**